ideascube/mediacenter/management/commands/remove_medias.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+import os
+import sys
+import argparse
+from os.path import join as pj
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from ideascube.mediacenter.models import Document
+from ideascube.utils import printerr
+
+
+class Command(BaseCommand):
+    help = ('Remove files from the mediacenter.')
+
+    def add_arguments(self, parser):
+        subs = parser.add_subparsers(
+            title='Commands', dest='cmd', metavar='',
+            parser_class=argparse.ArgumentParser)
+        remove = subs.add_parser(
+            'clean_orphans',
+            help='Clean mediacenter files not associated with a document.')
+        remove.set_defaults(func=self.clean_orphans)
```

Comment:
I had a feeling you'd want to add other cleanup operations, but I couldn't think of any, so I preferred asking. 😃 

Having subcommands here is fine, and it makes sense to keep the PR as it is in this regard.

However, do add a note about that in the commit message, please.

Also, I still think that `remove_medias` is not really a representative name for this command. It would be more representative of the new subcommand we'd add for #685 though.

So maybe just `clean`?

```
$ ideascube clean leftover-files
$ ideascube clean all-medias
```

They almost look like English sentences. 😛